backend/src/facebook-feed/facebook-feed.service.ts

```diff
@@ -24,7 +24,6 @@ export class FacebookFeedService {
     async fetchAndSaveFacebookFeeds() {
         this.logger.log('Fetching facebook feeds Started');
         FB.api(`${sources.vantaa}/posts`, { fields: queryParams })
-            .then(this.filterAlreadyExistingFeeds)
             .then(this.addProfileImage)
```

Comment:
This will also fail. Detailed comment given for the instagram service. Please refer.